apiTester/purchases.ts

```diff
@@ -162,3 +164,12 @@ function checkSyncObserverModeAmazonPurchase(productID: string,
   Purchases.syncObserverModeAmazonPurchase(
     productID, receiptID, amazonUserID, isoCurrencyCode, price);
 }
+
+function checkBeginRefundRequest(
+  entitlementInfo: PurchasesEntitlementInfo,
+  purchasesStoreProduct: PurchasesStoreProduct
+) {
+  Purchases.beginRefundRequestForActiveEntitlement((refundRequestStatus: REFUND_REQUEST_STATUS) => {}, errorCallback);
+  Purchases.beginRefundRequestForEntitlement(entitlementInfo, (refundRequestStatus: REFUND_REQUEST_STATUS) => {}, errorCallback);
+  Purchases.beginRefundRequestForProduct(purchasesStoreProduct, (refundRequestStatus: REFUND_REQUEST_STATUS) => {}, errorCallback);
```

Comment:
Nitpick. I guess we don't have line length linters right? This seems pretty long.